open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/admin/VirtualizerAdminResource.java

```diff
@@ -0,0 +1,148 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.virtualdataconnector.virtualiser.admin;
+
+import org.odpi.openmetadata.adminservices.OMAGServerAdminServices;
+import org.odpi.openmetadata.adminservices.OMAGServerOperationalServices;
+import org.odpi.openmetadata.adminservices.rest.VoidResponse;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Map;
+
+/**
+ * VirtualizerAdminResource provides the spring annotations for the server-side
+ * implementation of the administrative interface for
+ * the Virtualizer
+ */
+
+@RestController
+@RequestMapping("/open-metadata/admin-services/users/{userId}/servers/{serverName}")
+public class VirtualizerAdminResource{
+
+    private OMAGServerAdminServices       adminAPI            = new OMAGServerAdminServices();
+    private OMAGServerOperationalServices operationalServices = new OMAGServerOperationalServices();
+
+    /*
+     * =============================================================
+     * Help the client discover the type of the server
+     */
+
+
+    /**
+     * Return the origin of this server implementation.
+     *
+     * @return Server Origin
+     */
+    @RequestMapping(method = RequestMethod.GET, path = "/server-origin")
```

Comment:
The VirtualizerResource should only be adding the extra REST calls it needs to set up the VirtualizerConfig.
The general admin services should not be repeated here.